lasagne/layers/recurrent.py

```diff
@@ -438,6 +468,24 @@ class Gate(object):
     nonlinearity : callable or None
         The nonlinearity that is applied to the input gate activation. If None
         is provided, no nonlinearity will be applied.
+
+    References
```

Comment:
Sorry, my mistake for browsing refs while on Columbia's network.  The paper is "Learning to forget: continual prediction with LSTM" by Gers.  Using a forget gate of 5 is mentioned on page 15.